File: control-assurance-sp/src/types/GoDefForm.ts
import { IEntity } from "./Entity";
export interface IGoDefForm extends IEntity {
    Details?: string;
    Section1Title?: string;
    Section2Title?: string;
    Section3Title?: string;
    SummaryShortInstructions?: string;
    SummaryFullInstructions?: string;
    SummaryFormRatingText?: string;
    DGSignOffText?: string;
}

File: ControlAssuranceAPI/Models/GoDefForm.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class GoDefForm
{
    public int ID { get; set; }

    public string? Title { get; set; }

    public string? Details { get; set; }

    public string? Section1Title { get; set; }

    public string? Section2Title { get; set; }

    public string? Section3Title { get; set; }

    public string? SummaryShortInstructions { get; set; }

    public string? SummaryFullInstructions { get; set; }

    public string? SummaryFormRatingText { get; set; }

    public string? DGSignOffText { get; set; }

    public string? Access { get; set; }
}
